pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -393,8 +393,9 @@ type KubeVirtMetadata struct {
 }
 
 type AccessCredentialMetadata struct {
-	Succeeded bool   `xml:"succeeded,omitempty"`
-	Message   string `xml:"message,omitempty"`
+	Succeeded        bool   `xml:"succeeded,omitempty"`
+	Message          string `xml:"message,omitempty"`
+	DeprecatedMethod bool   `xml:"deprecatedMethod,omitempty"`
```

Comment:
There is no need for this. The launcher is able to push events. See `notifier`, what is left is to wire it to the `AccessCredentialManager`